.github/actions/setup-project/action.yml

```diff
@@ -60,3 +60,7 @@ runs:
     - name: 🛟 Install dependencies
       shell: bash
       run: pnpm install --frozen-lockfile
+
+    - name: Install wait-on plugin
```

Comment:
`setup-project` action is being used by all the workflows. Makes sense to install the dependency here and forget. Even if `wait-on` is not used in all the workflows the installation of it should be quick and is available as part of the tooling for the workflows.